src/current/cockroachcloud/cockroachdb-advanced-on-azure.md

```diff
@@ -1,73 +1,31 @@
-- [Customer Managed Encryption Keys (CMEK)]({% link cockroachcloud/cmek.md %})
-- [Egress Perimeter Controls]({% link cockroachcloud/egress-perimeter-controls.md %})
+- A cluster must have at minimum three nodes. A multi-region cluster must have at minimum three nodes per region. Single-node clusters are not supported on Azure.
+- The following [PCI-Ready]({% link cockroachcloud/pci-dss.md %}) and HIPAA features are not yet available on Azure. However, CockroachDB {{ site.data.products.advanced }} on Azure meets or exceeds the requirements of SOC 2 Type 2. Refer to [Regulatory Compliance in CockroachDB {{ site.data.products.advanced }}]({% link cockroachcloud/compliance.md %}).
+  - [Private Clusters]({% link cockroachcloud/private-clusters.md %})
+  - [Customer Managed Encryption Keys (CMEK)]({% link cockroachcloud/cmek.md %})
+  - [Egress Perimeter Controls]({% link cockroachcloud/egress-perimeter-controls.md %})
 
-## FAQs
+  You can configure IP allowlisting to limit the IP addresses or CIDR ranges that can access a CockroachDB {{ site.data.products.dedicated }} cluster on Azure, and you can use [Azure Private Link](https://learn.microsoft.com/azure/private-link/private-link-overview) to connect your applications in Azure to your cluster and avoid exposing your cluster or applications to the public internet. Refer to [Connect to your cluster]({% link cockroachcloud/connect-to-your-cluster.md %}#azure-private-link).
 
-The following sections provide more details about CockroachDB {{ site.data.products.dedicated }} on Azure.
+## Change data capture
 
-### Can CockroachDB {{ site.data.products.serverless }} clusters be deployed on Azure?
+CockroachDB {{ site.data.products.advanced }} supports [changefeeds](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/changefeed-messages), which allow your cluster to send data events in real-time to a [downstream sink](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/changefeed-sinks.html). [Azure Event Hubs](https://learn.microsoft.com/azure/event-hubs/azure-event-hubs-kafka-overview) provides an Azure-native service that can be used with a Kafka endpoint as a sink.
 
-CockroachDB {{ site.data.products.serverless }} is not currently available on Azure.
+## Disaster recovery
 
-### Can we use {{ site.data.products.db }} credits to pay for clusters on Azure?
+[Managed-service backups]({% link cockroachcloud/use-managed-service-backups.md %}?filters=advanced) automatically back up clusters in CockroachDB {{ site.data.products.cloud }}.
 
-Yes, a CockroachDB {{ site.data.products.cloud }} organization can pay for the usage of CockroachDB {{ site.data.products.dedicated }} clusters on Azure with {{ site.data.products.db }} credits. To add additional credits to your CockroachDB {{ site.data.products.cloud }} organization, contact your Cockroach Labs account team.
-
-### Can we migrate from PostgreSQL to CockroachDB {{ site.data.products.dedicated }} on Azure?
```

Comment:
@mdlinville I see that some of the removed content from here to the end of the file is actually moved to more meaningful sections, which looks good. Is there anything dropped outright? If so, worth updating the PR description to briefly mention what and why?

Reply:
Sure, let me explain. This page is about limitations of Dedicated (Advanced) on Azure, and we have removed limitations over time. Here I remove several items that are not limitations of Dedicated on Azure, but are just the way all the clouds work. I had so few items at the end of doing that, that I removed the FAQ section and the other subsections, in favor of just listing the few items that are still limitations on Azure or differences with Azure (for example, Azure has no support single-node clusters, and Azure currently does not support PCI-DSS.